Resources/Prototypes/_Lust/Arrows/Arrows.yml

```
@@ -0,0 +1,1906 @@
+  - CentralCommand
+  - MTO
+  - MTOL
+
+#РАБОТА СОЛДАТ
+- type: job
+  id: MTS
+  name: job-name-MTO
+  playTimeTracker: MTS
+  icon: "JobIconArrows"
+  accessGroups:
+  - AllAccess
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)